.env.test

```diff
@@ -10,3 +10,4 @@ FAVICON_URL='https://edx-cdn.org/v3/default/favicon.ico'
 FEATURE_FILE_ATTACHMENT=true
 ENTERPRISE_SUPPORT_URL = ''
 ENTERPRISE_SUPPORT_REVOKE_LICENSE_URL = ''
+TABLEAU_URL='https://enterprise-tableau.edx.org'
```

Comment:
do we really need to add a real url here? can we use some fake url?

Reply:
The address is already public in DNS records. I donot see any reason to hide it.